src/i18n/locales/en-US.json

```diff
@@ -67,7 +67,7 @@
   "components.send.amountfield.label": "Amount",
   "components.send.biometricauthscreen.DECRYPTION_FAILED": "Fingerprint login failed. Please use an alternate login method.",
   "components.send.biometricauthscreen.NOT_RECOGNIZED": "Fingerprint was not recognized. Try again",
-  "components.send.biometricauthscreen.SENSOR_LOCKOUT": "You used too many fingers. Sensor is disabled",
```

Comment:
Are we sure this changed meaning is accurate?

Reply:
Yes, he checked with me before changing it.